testsuite/MDAnalysisTests/analysis/test_diffusionmap.py

```diff
@@ -0,0 +1,80 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4 fileencoding=utf-8
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver
+# Beckstein and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+from __future__ import print_function
+import numpy as np
+import MDAnalysis
+import MDAnalysis.analysis.diffusionmap as diffusionmap
+from numpy.testing import (assert_almost_equal, assert_equal, raises)
+
+
+from MDAnalysisTests.datafiles import PDB, XTC
+
+
+class TestDiffusionmap(object):
+    def __init__(self):
```

Comment:
Please switch to `setUp` instead of init. We might change to py.test soonish and py.test doesn't run Test classes that have a `__init__` function. Lets prepare new code for the future directly.